addons/mail/static/src/new/web/messaging_menu/messaging_menu.xml

```diff
@@ -79,7 +79,9 @@
                             <i class="fa fa-mail-reply me-1"/>You:
                         </t>
                         <t t-else="">
-                            <t t-esc="preview.mostRecentMsg.author?.name"/>:
+                            <t t-if="preview.mostRecentMsg.author">
+                                <t t-esc="preview.mostRecentMsg.author.name"/>:
+                            </t>
```

Comment:
What was happening with the `t-esc` if the author was undefined before this change?

Reply:
Oh I see it's for the `:`, but then I should suggest to have `else + if` `elif` instead

Reply:
In messages without  author (for example nonusers in live chat) the `:` symbol remains without author name. Actually if there is not author name we don't want `:` either.